index.js

```diff
@@ -13,7 +13,7 @@ require('bootstrap/js/popover')
 // 1. creating issue
 // 2. creating pr
 // 3. comment issue, comment pr, inline pr comment
-const textareaSelectorGithub = ['[name="issue[body]"]', '[name="pull_request[body]"]', '[name="comment[body]"]']
+const textareaSelectorGithub = ['[name="issue[body]"]', '[name="pull_request[body]"]', '[name="comment[body]"]', '[name="pull_request_review[body]"]']
```

Comment:
We should split this up into multiple lines IT'S OVER 80 CHARACTERS AND THE WEB DIES WHEN YOU DO THAT.

Na, it seems good to me ![](https://media2.giphy.com/media/wpoLqr5FT1sY0/giphy.gif)